core/src/main/java/org/frankframework/stream/ThreadConnector.java

```diff
@@ -86,7 +86,7 @@ public <R> R startThread(R input) {
 		}
 		if (childThread != parentThread) {
 			childThread.setName(parentThread.getName() + "/" + childThread.getName());
-			IbisMaskingLayout.addToThreadLocalReplace(hideRegex);
+			IbisMaskingLayout.setThreadLocalReplace(hideRegex);
```

Comment:
Eigenlijk zou dit niet meer hoeven toch? De adapter en receiver vullen de threadlocal aan en verwijderen zelf ook na het verwerken van het bericht.

Reply:
Dit is juist nodig! Dit is een nieuwe thread, en dit zorgt dat de hide-regexes van de ene thread worden overgezet naar de andere thread bij het aanroepen van een sub-adapter.